app/views/layouts/_navbar.html.erb

```diff
@@ -0,0 +1,39 @@
+<head>
+  <script src="http://code.jquery.com/jquery-1.11.0.min.js"></script>
```

Comment:
The JQuery reference is no longer required, so you can remove it.

Reply:
Ok! understood!